tidy3d/components/simulation.py

```diff
@@ -69,6 +71,9 @@
 # maximum number of sources
 MAX_NUM_SOURCES = 1000
 
+# maximum number of lumped elements
+MAX_NUM_LUMPED_ELEMENTS = 65530
```

Comment:
I wonder if it's necessary to bound the maximal number of elements. I think this number 65530 is coming from the upper limit of the number of materials in the simulation, and is validated [here](https://github.com/flexcompute/tidy3d/blob/722ebe641a0faaff6cd6f56cb2349a3d5adc7430/tidy3d/components/scene.py#L78-L92). Or do I miss anything? If so, I'm not sure if we should move the `lumped_elements` field to scene.py or add a validator in simulation to count the total number of medium for structures + lumped_elements. The latter is easier I think.